setup.py

```diff
@@ -29,6 +29,8 @@
         'License :: OSI Approved :: BSD License',
         'Topic :: Software Development :: Libraries :: Python Modules'],
 
+    install_requires=("django_apptemplates",),
```

Comment:
This is not the right dependency :)
And I don't want to force installation of packages, the README is explicit enough